lib/src/entities/address.dart

```diff
@@ -1,6 +1,13 @@
 class Address {
-  String name;
-  String mailAddress;
+  final String name;
+  final String mailAddress;
```

Comment:
I can't add this change with a minor version bump.
Please revert this.  I'll try to keep this change in mind for a future version.